src/commands/realType.ts

```diff
@@ -52,8 +43,7 @@ export async function realType(text: string, options: RealTypeOptions = {}) {
     }, [] as string[]);
 
   for (const char of chars) {
-    assertChar(char);
-    await realPress(char, {
```

Comment:
I've refactored this to use `rawRealPress` (that I've introduced in this PR) since I believe that "composite" commands should use "raw" commands over the ones that are exposed publicly as Cypress commands